internal/resources/client.go

```diff
@@ -10,7 +11,16 @@ import (
 	"github.com/launchdarkly/ldcli/internal/errors"
 )
 
+type UnauthenticatedClient interface {
```

Comment:
We can use this interface in the login tests so we don't need to implement an unused `MakeRequest` method.